docs/registry/index.rst

```diff
@@ -336,6 +336,44 @@ and then you can run:
   {'flashheros.data': <Table name="flashheros.data">... 29 columns ...</Table>, 'ivoa.obscore': <Table name="ivoa.obscore">... 0 columns ...</Table>}
 
 
+Alternative Registries
+======================
+
+There are several RegTAP services in the VO.  PyVO by default uses the
+one at the TAP access URL http://reg.g-vo.org/tap.  You can use
+alternative ones, for instance, because they are nearer to you or
+because the default endpoint is down.
+
+You can pre-select the URL by setting the ``IVOA_REGISTRY`` environment
+variable to the TAP access URL of the service you would like to use.  In
+a bash-like shell, you would say::
+
+  export IVOA_REGISTRY="http://vao.stsci.edu/RegTAP/TapService.aspx"
```

Comment:
I know that you don't yet want review, so I leave only one comment:

we should be able to do this with a more programatic way, from inside the python interpreter.
Whether to use the astropy config system, or also provide ways to override the defaults in a session, or be able to do both is a detail, but I very strongly think it should be doable without shell variables.